website/sidebars.json

```diff
@@ -9,7 +9,8 @@
       "concepts/gas",
       "concepts/indexer",
       "concepts/networks",
-      "concepts/transaction"
+      "concepts/transaction",
+      "concepts/storage-solutions"
```

Comment:
This looks a little confusing in the sidebar as we have multiple sections that talk about storage. As a new user, I think I would be a little lost when trying to navigate this. Let's have a sub-menu for "Data Storage" that has two parts. "Storage on NEAR" and "Alternative Solutions" or something like this.

Reply:
I've grouped the articles together, under a "Data Storage" category:
- Data Storage
   - Storage on NEAR
   - Storage Staking
   - Alternative Solutions